code/game/objects/structures/transit_tubes/transit_tube.dm

```diff
@@ -12,6 +12,7 @@
 	var/list/tube_dirs //list of directions this tube section can connect to.
 	var/exit_delay = 1
 	var/enter_delay = 0
+	var/time_to_unwrench = 20 //deciseconds
```

Comment:
* use `2 SECONDS` instead of 20 to avoid an unnecessary comment like this
* use `var/const/time_to_unwrench` or a define as you're never modifying this var

Reply:
thank